### PR TITLE
refactor: Apply `webpack-defaults`

### PR DESCRIPTION
**Pull request by d3viant0ne**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "useBuiltIns": true,
+        "targets": {
+          "node": "4.3"
```
Comment (michael-ciniawsky) [inline .babelrc]: `4.8`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+export default function (content) {
+  this.value = content;
```
Comment (michael-ciniawsky) [inline src/index.js]: Only needed in `webpack =< v1.0.0`

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  const sanitizeUnicode = JSON.stringify(content)
```
Comment (michael-ciniawsky) [inline src/index.js]: Just `sanitized/escaped` ?

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    .replace(/\u2028/g, '\\u2028')
+    .replace(/\u2029/g, '\\u2029');
+  return `export default ${sanitizeUnicode}`;
```
Comment (michael-ciniawsky) [inline src/index.js]: This should definitely happen, but breaks for Angular folks (`css-loader` again 😞 )

**webpack.config.js**
```js
{
   test: /\.css$/,
   use: [ 'raw-loader', 'css-loader' /* <= Needs module.exports = '' to work */ ]
}
```

The alternative is to use `postcss-loader` standalone and ditch `raw-loader`, but also not a 💯  replacement either

Comment (d3viant0ne) [reply src/index.js]: Yeah, I spent most of my eventing trying to find a way around this in the CLI without much success.

For the time being, I think we are just going to have to leave this alone & tackle the es6 export after the fact.

**Pull request merged**